waspc/src/Wasp/Generator/ExternalConfig/PackageJson.hs

```diff
@@ -0,0 +1,56 @@
+module Wasp.Generator.ExternalConfig.PackageJson
+  ( validatePackageJson,
+  )
+where
+
+import qualified Data.Map as M
+import qualified Wasp.ExternalConfig.PackageJson as P
+import Wasp.Generator.Common (prismaVersion)
+import Wasp.Generator.WebAppGenerator.Common (reactRouterVersion)
+
+validatePackageJson :: P.PackageJson -> IO [String]
```

Comment:
Let's maybe add `type ErrorMsg = String` here.

Reply:
One thought on top of this: often, in our code, we just use Strings as error messages. General advice is that this is not good, you should use more structured types, but the truth is, for our needs this seems to be right at the moment.
However, it is still much nicer to see `ErrorMsg` in output type than `String` because it tells me what it is, which is why `type ErrorMsg = String`.
Now, sometimes it is nice to call it more specifically: `type CompilerError = String`, or `type ParsingError = String`, but in many parts of the code we don't want to bother iwth that and just doing `type ErrorMsg = String` is enough. But that means we have a lot of places where we do the same thing: define `type ErrorMsg = String`. So I wonder if we should just have central `type ErrorMsg = String` in some `Wasp.Utils` or something like that? We don't have to do this in this PR, but wanted to mention this now since I was talking about this. What do you think?

Reply:
I'll turn this line of thinking into an issue 👍

Reply:
https://github.com/wasp-lang/wasp/issues/2310